tools/debian_buildenv.sh

```diff
@@ -3,12 +3,14 @@
 # shellcheck disable=SC1091
 set -o pipefail
 
+CALL=$0
```

Comment:
Do we really need to create an alias for $0?

Reply:
Because of the following shift call.
Do you know why it is there?

Reply:
Possibly a leftover from a prevous version of the code.

Reply:
should we try to remove it with this PR?

Reply:
Yes, please try